torch_lr_finder/lr_finder.py

```diff
@@ -460,6 +463,19 @@ def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None):
 
         # Plot loss as a function of the learning rate
         ax.plot(lrs, losses)
+
+        # Plot the suggested LR in a one-time loop
+        while suggestion:
+            try: 
+                mg = (np.gradient(np.array(losses))).argmin()
+            except:
+                print("Failed to compute the gradients, there might not be enough points.")
+                break
+
+            print(f"Min numerical gradient: {lrs[mg]:.2E}")
+            ax.plot(lrs[mg], losses[mg], markersize=10, marker='o', color='red')
+            break
```

Comment:
```suggestion
                ax.plot(lrs[mg], losses[mg], markersize=10, marker='o', color='red')
```

Reply:
@davidtvs how about now? I summarize the **pseudo-code** (I'm afraid it is not clear in the `GitHub code review` mode):

```python
if suggest_lr:
    try:
        mg = np.gradient(XXX)
    except:
        print ('error')
        mg = None
    if mg is not None:
        print ('Suggest LR is XXX')
        ax.plot(XXX)
```

---

Maybe in the future we can implement some functions for different `suggest_lr` algorithms? 
```python
if suggest_lr == 'steepest':
    self._plot_steepest(history, ax)
elif suggest_lr == 'XXX'
    self._plot_XXX(history, ax)
else:
    print ('Oops!')
```